Dockerfile

```diff
@@ -0,0 +1,114 @@
+FROM tomcat:9-jdk11-openjdk as mother
+LABEL maintainer="Alessandro Parma<alessandro.parma@geo-solutions.it>"
+
+RUN apt-get update && apt-get install -y unzip
+
+# accepts local files and URLs. Tar(s) are automatically extracted
+WORKDIR /output/datadir
+ARG GEOSERVER_DATA_DIR_SRC="./.placeholder"
+ADD "${GEOSERVER_DATA_DIR_SRC}" "./"
+
+# accepts local files and URLs. Tar(s) are automatically extracted
+WORKDIR /output/webapp
+ARG GEOSERVER_WEBAPP_SRC="./.placeholder"
+ADD "${GEOSERVER_WEBAPP_SRC}" "./"
+
+# zip files require explicit extracion
+RUN \
+    if [ "${GEOSERVER_WEBAPP_SRC##*.}" = "zip" ]; then \
+        unzip "./*zip"; \
+        rm ./*zip; \
+    fi \
+    && [ -d "./geoserver" ] || (mkdir -p ./geoserver && unzip ./geoserver.war -d ./geoserver && rm ./geoserver.war)
+
+RUN apt-get update; apt-get upgrade --yes; apt-get install wget --yes
+RUN wget https://downloads.sourceforge.net/project/libjpeg-turbo/1.5.3/libjpeg-turbo-official_1.5.3_amd64.deb && dpkg -i ./libjpeg*.deb && apt-get -f install
```

Comment:
Would you recommend to always build in/use the libjpeg-turbo extension and lib?

Reply:
I'd remove all extensions from the official images by default